data/ecosystems/s/sui.toml

```diff
@@ -383,8 +384,8 @@ url = "https://github.com/MystenLabs/snarkjs"
 url = "https://github.com/MystenLabs/solvency-proofs"
 
 [[repo]]
-url = "https://github.com/MystenLabs/sui"
 tags = ["Protocol"]
+url = "https://github.com/MystenLabs/sui"
```

Comment:
can you keep the tags parameter below the URL like it was originally? not sure why it was changed